src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -3,13 +3,62 @@
 
 #include "pch.h"
 #include "Profiles.h"
+
 #include "PreviewConnection.h"
 #include "Profiles.g.cpp"
 #include "EnumEntry.h"
 
 #include <LibraryResources.h>
 #include "..\WinRTUtils\inc\Utils.h"
 
+// This function is a copy of DxFontInfo::_NearbyCollection() with
+// * the call to DxFontInfo::s_GetNearbyFonts() inlined
+// * checkForUpdates for GetSystemFontCollection() set to true
+static wil::com_ptr<IDWriteFontCollection1> NearbyCollection(IDWriteFactory* dwriteFactory)
+{
+    // The convenience interfaces for loading fonts from files
+    // are only available on Windows 10+.
+    wil::com_ptr<IDWriteFactory6> factory6;
+    // wil's query() facilities don't work inside WinRT land at the moment.
```

Comment:
Is this filed against microsoft/wil and/or are we just behind on updating our copy of wil?

Reply:
I got a TODO item to open a PR for that when I'm back at work. Since it's an easy fix that seemed better to me than opening an issue. 🙂

Reply:
IIRC that's an ordering issue - something about including wil _before_ winrt, or vice-versa, or something like that. My memory's hazy on this...